### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by talrejah02**
Please review, suggestion's are welcomed.

### CODE REVIEW

File: components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+@import url(../common-css/drawer.css);
+@import url(../common-css/container.css);
+@import url(../../style.css);
+
+body{
+    background-color: #ffff;
+    margin: 0;
+    
+    
+}
```
Comment (ScoobyDubeyDo) [inline components/Avatar/avatar.css]: please avoid applying styles to the body itself.

File: components/common-css/container.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    margin:0;
+}
+code{
+margin: 1rem;
+margin-top: 0;
+margin-bottom: 0;
+background-color: var(--primary);
+font-size: 0.8rem;
+border-radius: 7px;
+padding-left: 1rem !important;
```
Comment (ScoobyDubeyDo) [inline components/common-css/container.css]: Try avoid using !important

**Review by ScoobyDubeyDo — CHANGES_REQUESTED** [review]

File: components/Avatar/avatar.css (hunk 1 above)
Comment (talrejah02) [reply components/Avatar/avatar.css]: okay sure thanks

**Review by manojkumar8400 — APPROVED** [review]
looks good.

File: components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+@import url(../../docs-css/reset.css);
+@import url(../common-css/drawer.css);
+@import url(../common-css/container.css);
+@import url(../../style.css);
+
+body{
+    margin: 0;
+}
+
```
Comment (kshitijthatte) [inline components/Avatar/avatar.css:9]: As recommended by @aman-can , avoid using style directly on HTML tags. Also, you've already styled body in style.css

**Review by kshitijthatte — APPROVED** [review]

**Pull request merged**